src/lib.rs

```diff
@@ -277,97 +208,111 @@ impl UPClientZenoh {
         };
         Ok(uattributes)
     }
+
+    /*        The table for mapping resource ID to message type
```

Comment:
If this table doesn't exist in up-spec it'd be great to have you contribute to adding it there!

Reply:
So is the table clear to you? The reason why I hesitate to add the table into spec is that I'm afraid it makes readers difficult to understand.

Reply:
Yeah, I get it. I spent a few minutes just mentally going over the table to see if it aligned with my understanding.

I think, even if it takes some time for someone to process what's in the table, it makes a lot more sense to have it written up in a place like up-spec that can be referenced for any implementation :slightly_smiling_face: